src/app/public/modules/date-range-picker/types/date-range-calculator-config.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from './date-range-calculator-validate-function';
 
 /**
- * @internal
+ * The configuration for a date range calculator.
```

Comment:
Since we have a description for this interface, should we have something similar for `SkyDateRangeCalculation`? Not sure if we need it, but just checking to see if we should have a description for consistency and/or to explain the purpose of this interface vs. the other one.